.travis.yml

```diff
@@ -33,13 +33,13 @@ jobs:
         - make .gitvalidation
         - make binaries
         - make containerd
-      go: 1.9.x
+      go: "1.9.x"
     - script:
         - make install.tools
         - make .gitvalidation
         - make binaries
         - make containerd
-      go: tip
+      go: "1.10"
```

Comment:
Don't we need ".x"?

Reply:
Not yet, synching with containerd tested golang versions.